app/models/manageiq/providers/lenovo/physical_infra_manager.rb

```diff
@@ -19,4 +19,12 @@ def self.ems_type
   def self.description
     @description ||= "Lenovo XClarity"
   end
+
+  # Returns a new connection to the LXCA
+  def connection
+    connect(:user => authentications.first.userid,
+            :pass => authentications.first.password,
+            :host => endpoints.first.hostname,
+            :port => endpoints.first.port)
```

Comment:
If all this does is allow you to call connect with the default values why not just call connect and let it use the defaults it already has?
It has the same defaults for all of these, https://github.com/ManageIQ/manageiq-providers-lenovo/blob/master/app/models/manageiq/providers/lenovo/manager_mixin.rb#L18-L21